spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+describe SessionsController do
+  describe "GET new" do
+
+    it "renders the new template for unauthenticaed users" do
```

Comment:
I would not test this. Especially because you are in a new action and testing render on a new template. You have not written code for this, so you should not test it.